packages/main/src/Slider.js

```diff
@@ -0,0 +1,457 @@
+		 * @public
+		 */
+		step: {
+			type: Float,
+			defaultValue: 1,
+		},
+		/**
+		 * Current value of the slider
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		value: {
+			type: Float,
+			defaultValue: 0,
+		},
+		/**
+		 * Determines end point of a selection - position of a second handle on the slider. Allow defining of a <b>selected range</b>.
+		 * When <code>endValue</code> is specified, the <code>value</code> property is used as a "start value" to determine a selected range on the slider.
+		 * If the values is lower/higher than the allowed minimum/maximum, they will be set to the corresponding min/max values of the slider.
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		 endValue: {
+			type: Float,
+		},
+		/**
+		 * Enables tick marks visualization for each step. The step value must not be set to 0
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		 tickmarks: {
+			type: Boolean,
+		},
+		/**
+		 * Defines whether the <code>ui5-slider</code> is in disabled state.
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines the text of the <code>ui5-range-slider</code>.
+		 * <br><b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+		 *
+		 * @type {Node[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: Node,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fired when the value changes and the user has finished interacting with the slider.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+		/**
+		 * Fired when the value changes due to user interaction.
+		 *
+		 * @event
+		 * @public
+		*/
+		input: {},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/Label";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.Slider
+ * @extends sap.ui.webcomponents.base.UI5Element
+ * @tagname ui5-range-slider
+ * @public
+ */
+class Slider extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get template() {
+		return SliderTemplate;
+	}
+
+	static get styles() {
+		return SliderStyles;
+	}
+
+	static async onDefine() {
+		await Promise.all([
+			fetchI18nBundle("@ui5/webcomponents"),
+		]);
+	}
+
+	constructor() {
+		super();
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	_handleChange() {
+
+	}
+
+	_handleInput() {
+
+	}
+
+	// TODO Include touch/pointer events, keyboard handling
+	/**
+	 * Called when the user starts interacting with the slider
+	 */
+	_handleDown(event) {
+		const _this = this;
```

Comment:
don't need _this, if you have it because of:
```js
const moveHandler = moveEvent => {
	_this._handleMove(moveEvent);
};
```

note arrow functions do not default this to the window scope, rather they execute in the scope they are created

Reply:
Yup :) converted the functions to arrow ones and forgot to remove the local this ref, thanks :)